src/intro-to-c/enums/enum-practice-answers.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Enums
+author: Matthew Gibbons
+---
+
+@. You are a half decent farmer in a video game that received a handful of seeds in the mail. With them came a note that informed you they all take the same amount of time to grow. You decided to plant them all and nurture them to maturity. The day has come to harvest, and you need to keep track of the crops you recognize. Write a program that will keep a count of each crop you have harvested, including unknowns, and print the number harvested in order of experience gain (least to greatest). Unknown crops can be last on the list. Below are code snippets so an answer key can exist.
+
+``` c
+// Random seed
+srand(24164132);
+// Random number generated
+1 + rand() % 13;
+// Crop Values
+3, 5, 6, 9, 11, 12;
+```
```

Comment:
If it will use elements of randomness, I would explicitly write in the question that you will also need to generate random crops, and perhaps which crops are the ones that are being farmed.